src/fidesops/service/saas_request/override_implementations/mailchimp_request_overrides.py

```diff
@@ -0,0 +1,136 @@
+from json import dumps
+from typing import Any, Dict, List
+
+import pydash
+from requests import get, put
+
+from fidesops.common_exceptions import ClientUnsuccessfulException, ConnectionException
+from fidesops.core.config import config
+from fidesops.graph.traversal import TraversalNode
+from fidesops.models.policy import Policy
+from fidesops.models.privacy_request import PrivacyRequest
+from fidesops.service.saas_request.saas_request_override_factory import (
+    SaaSRequestOverrideFactory,
+    SaaSRequestType,
+)
+from fidesops.util.collection_util import Row
+
+
+@SaaSRequestOverrideFactory.register(
```

Comment:
I like that we can keep all the overrides for a specific connector in a single place 👍

Reply:
just noting here that we'll try to import the method such that you can just call `@register`